Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/Microsoft.Toolkit.Forms.UI.Controls.csproj

```diff
@@ -2,33 +2,24 @@
 
   <PropertyGroup>
     <TargetFramework>net462</TargetFramework>
-    <RootNamespace>Microsoft.Toolkit.Win32.UI.Controls</RootNamespace>
-    <AssemblyName>Microsoft.Toolkit.Win32.UI.Controls</AssemblyName>
+    <RootNamespace>Microsoft.Toolkit.Forms.UI.Controls</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Forms.UI.Controls</AssemblyName>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <RunCodeAnalysis>true</RunCodeAnalysis>
 
-    <Title>Windows Community Toolkit Win32 Controls</Title>
-    <Description>This library provides XAML user controls for Win32. It is part of the Windows Community Toolkit.</Description>
-    <PackageTags>WebView Win32 WPF WindowsForms</PackageTags>
-
-    <!-- Need Metadata contract 6.0 or later -->
-    <!-- First introduced in 17110: Insider Preview (Fast) -->
-    <!-- Most people will have RTM 17134: April 2018 Update -->
-    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
-    <!--
-    When changing this value, ensure the SDK is installed with the build process.
-      - /build/vsts-build.yml
-
-    This also needs to be installed on your local machine. Can do this with PowerShell:
-      ./build/Install-WindowsSDKISO.ps1 17733
-    -->
-    <TargetPlatformVersion>10.0.17733.0</TargetPlatformVersion>
+    <Title>Windows Community Toolkit Windows Forms Controls</Title>
+    <Description>This library provides XAML user controls for Windows Forms. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>WebView Win32 WindowsForms</PackageTags>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS;NET47;DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>
```

Comment:
Should be rewritten to `<DefineConstants>$(DefineConstants);DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>`